main.go

```diff
@@ -124,6 +124,14 @@ func getAgentConfigPathFromHome() string {
 	return path.Join(home, "wiresteward.json")
 }
 
+func getAgentTokFilePathFromHome() string {
```

Comment:
```suggestion
func getAgentTokenFilePathFromHome() string {
```

Unless there's some convention here I'm unaware of, I think using `Token` is clearer.

Reply:
Also "get" is very anti-pattern in Go